Java/EggDroppingPuzzle.java

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
You are deleting one file from the Java Directory, "CoinChange" Kindly resolve that